### DISCUSSION TITLE
Adding an "Edit this Page" with GitHub icon to html Table of Contents

### DISCUSSION THREAD

**Discussion by ran-codes** [quarto-dev/quarto-cli#1821] [Q&A] [answered]
Having GH links below table of contents is very helpful. I have snipped an example from the Quarto website. But I have been unable to find out how to add this via looking at the quarto.org documentation and the quarto-web source code. Any suggestions? Thanks!

![image](https://user-images.githubusercontent.com/45013044/184155424-dfc6e455-1d38-449f-a289-44d47c6939b6.png)

**Comment by jjallaire** [comment, marked as answer]
https://quarto.org/docs/websites/website-navigation.html#github-links
